Zend/zend_weakrefs.c

```diff
@@ -0,0 +1,235 @@
+/*
+   +----------------------------------------------------------------------+
+   | Zend Engine                                                          |
+   +----------------------------------------------------------------------+
+   | Copyright (c) 1998-2018 Zend Technologies Ltd. (http://www.zend.com) |
```

Comment:
Nit: will we follow the same for Zend's Copyright as we [did for PHP's](https://github.com/php/php-src/commit/38c337f2)?

Reply:
Fixed ...